### ISSUE TITLE
Doesn't write scalars for two models of the same framework

### ISSUE THREAD

**Issue by dev-rinchin** [allegroai/clearml#767]
let’s add second catboost model training to [catboost_example.py](https://github.com/allegroai/clearml/blob/master/examples/frameworks/catboost/catboost_example.py):

```
...
    catboost_model = CatBoostRegressor(iterations=iterations, verbose=False)
    catboost_model2 = CatBoostRegressor(iterations=iterations+200, verbose=False)
...
    catboost_model.fit(train_pool, eval_set=test_pool, verbose=True, plot=False, save_snapshot=True)
    catboost_model2.fit(train_pool, eval_set=test_pool, verbose=True, plot=False, save_snapshot=True)
...
```

as result, "scalars" tab in CML web shows just one learn and one validation plots, I was supposed to get two. It looks like the first plot was overwritten by the second one.

<img width="1013" alt="Screenshot 2022-09-05 at 13 31 25" src="https://user-images.githubusercontent.com/57899558/188484011-e6f1529a-2d45-4550-b560-2add57ae03bf.png">

## To reproduce
run multiple models training of any supported framework

## Expected behaviour
plots for both models

## Environment
* Server type: self hosted
* ClearML SDK Version: 1.6.4
* ClearML Server Version: WebApp: 1.6.0-213 • Server: 1.6.0-213 • API: 2.20
* Python Version: 3.7
* OS: Linux

## Related Discussion
[(https://clearml.slack.com/archives/CTK20V944/p1661358596687939)](https://clearml.slack.com/archives/CTK20V944/p1661358596687939)

**Comment by pollfly** [comment]
Hey @dev-rinchin ! Just letting you know that this issue has been resolved in the recently released [v1.10.0](https://github.com/allegroai/clearml/releases/tag/v1.10.0). Let us know if there are any issues :)

**Comment by jkhenning** [comment]
@dev-rinchin closing this, please reopen if required.
